From d747e41825135387dfe9930184525316a1e495d6 Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Mon, 12 Aug 2019 14:39:05 +0700
Subject: [PATCH 310/405] dt-bindings: timer: renesas: tmu: Document r8a774b1
 TMU

Document the RZ/G2N SoC in the Renesas TMU bindings.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 Documentation/devicetree/bindings/timer/renesas,tmu.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/timer/renesas,tmu.txt b/Documentation/devicetree/bindings/timer/renesas,tmu.txt
index 48d6dc5..efd544d 100644
--- a/Documentation/devicetree/bindings/timer/renesas,tmu.txt
+++ b/Documentation/devicetree/bindings/timer/renesas,tmu.txt
@@ -11,6 +11,7 @@ Required Properties:
   - compatible: must contain one or more of the following:
     - "renesas,tmu-r8a7740" for the r8a7740 TMU
     - "renesas,tmu-r8a774a1" for the r8a774a1 TMU
+    - "renesas,tmu-r8a774b1" for the r8a774b1 TMU
     - "renesas,tmu-r8a774c0" for the r8a774C0 TMU
     - "renesas,tmu-r8a7778" for the r8a7778 TMU
     - "renesas,tmu-r8a7779" for the r8a7779 TMU
-- 
2.7.4

